genome.spec

```diff
@@ -75,7 +89,7 @@ typedef structure {
 } InferenceInfo;
 
 /*
-Structure for a single feature CDS
+Structure for a single coding sequence.
 
       flags are flag fields in GenBank format. This will be a controlled vocabulary.
         Initially Acceptable values are pseudo, ribosomal_slippage, and trans_splicing
```

Comment:
CDS needs docs

Reply:
I expanded the docs here. I didn't cover everything, but added a lot. Feel free to review if you have time.